### PR TITLE
GHA: Add debian builds to custom repo

### PR DESCRIPTION
**Pull request by d-513**
Before merge add required secrets:
- [ ] GPG_PRIVATE_KEY - the gpg key used for signing packages
- [ ] GPG_KEY_ID - the gpg key id for signing packages
- [ ] SSH_PRIVATE_KEY - ssh key for pushing to repo via polymcbot

TODO:
- [X] Debian repo and CI integration
- [ ] Instructions for install
- [ ] Permanent domain for the apt repo (apt.polymc.org)

### CODE REVIEW

**Comment by txtsd** [comment]
Why change the quotes to double quotes?

**Comment by d-513** [comment]
> Why change the quotes to double quotes?

accident when I was messing with the workflows. I can change it back

File: .github/workflows/debian.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          echo 'deb [signed-by=/usr/share/keyrings/makedeb-archive-keyring.gpg arch=all] https://proget.hunterwittenborn.com/ makedeb main' | \
+          sudo tee /etc/apt/sources.list.d/makedeb.list
+          sudo apt update
+          sudo apt install makedeb
+      - name: Clone MPR Repo
+        run: git clone ssh://mpr@mpr.makedeb.org/polymc.git packages/debian
+      - name: Update Version
+        run: |
+          sed -i 's/pkgver=.*/pkgver=${{ env.VERSION }}/' packages/debian/PKGBUILD
+          sed -i 's/pkgrel=.*/pkgrel=1/' packages/debian/PKGBUILD
```
Comment (txtsd) [inline .github/workflows/debian.yml:43]: What happens when pkgrel needs to be bumped?

Comment (d-513) [reply .github/workflows/debian.yml:43]: This only runs on new releases. If we need to bump pkgrel, we do it in MPR. Next release it gets reset to 1.

Comment (txtsd) [reply .github/workflows/debian.yml:43]: Okay makes sense.

**Comment by txtsd** [comment]
Just, uh, unchange the quotes and I'll approve.

**Comment by d-513** [comment]
Yes I’ll do it when I’m home

**Comment by txtsd** [comment]
Looks good. Just install instructions left now?

**Comment by d-513** [comment]
Yes and most likely finding a suitable host as it seems wrong to have this on pages

**Comment by ZekeZDev** [comment]
I think we should probably leave this for 1.1.1 and encourage usage of mpr, appimage or flatpak

**Comment by d-513** [comment]
For now MPR is fine, and there doesn’t seem to be many static hosts that would allow us to do this

**Pull request closed without merging**